pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -118,6 +118,11 @@ def test_moral_graph_with_edge_present_over_parents(self):
             self.assertTrue(edge in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')] or
                             (edge[1], edge[0]) in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')])
 
+    
+    def test_local_independencies(self):
+        from pgmpy.independencies import Independencies
+        self.assertEqual(self.G.local_independencies('a'),Independencies(['a',['b','c']]))
```

Comment:
Yeah, go ahead with

On Wed, Oct 28, 2015 at 11:50 PM, Kshitij Saraogi <notifications@github.com>
wrote:

> In pgmpy/tests/test_models/test_BayesianModel.py
> <https://github.com/pgmpy/pgmpy/pull/498#discussion_r43296926>:
>
> > @@ -118,6 +118,11 @@ def test_moral_graph_with_edge_present_over_parents(self):
> >              self.assertTrue(edge in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')] or
> >                              (edge[1], edge[0]) in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')])
> >
> > +
> > +    def test_local_independencies(self):
> > +        from pgmpy.independencies import Independencies
> > +        self.assertEqual(self.G.local_independencies('a'),Independencies(['a',['b','c']]))
>
> @ankurankan <https://github.com/ankurankan> I think adding an __eq__
> method in the Independencies class would be a better choice, since it can
> be used in the future.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pgmpy/pgmpy/pull/498/files#r43296926>.
>
> --
> You received this message because you are subscribed to the Google Groups
> "pgmpy" group.
> To unsubscribe from this group and stop receiving emails from it, send an
> email to pgmpy+unsubscribe@googlegroups.com.
> To post to this group, send email to pgmpy@googlegroups.com.
> To view this discussion on the web visit
> https://groups.google.com/d/msgid/pgmpy/pgmpy/pgmpy/pull/498/r43296926%40github.com
> <https://groups.google.com/d/msgid/pgmpy/pgmpy/pgmpy/pull/498/r43296926%40github.com?utm_medium=email&utm_source=footer>
> .
>
> For more options, visit https://groups.google.com/d/optout.
>